lib/hop_swipe_cards.dart

```diff
@@ -417,43 +414,43 @@ typedef CurrentIndexInDisplay = void Function(
 
 class CardAnimation {
   static Animation<Alignment> frontCardAlign(
-      {double currentXLocation,
-      AnimationController controller,
-      Alignment currentAlignment,
-      Alignment baseAlign,
-      double swipeEdge,
-      bool pointLeft,
-      bool allowSwipeUpAndDown,
-      int swipedCards,
-      bool preventLeftSwipe,
-      Alignment afterSwipeAlignment}) {
+      {double? currentXLocation,
+      AnimationController? controller,
+      Alignment? currentAlignment,
+      Alignment? baseAlign,
+      double? swipeEdge,
+      bool? pointLeft,
+      bool? allowSwipeUpAndDown,
+      int? swipedCards,
+      bool? preventLeftSwipe,
+      Alignment? afterSwipeAlignment}) {
```

Comment:
- Why are these properties that are bool type able to be null?

Reply:
`bool? preventLeftSwipe,` can be null because _onRestrictLeftSwipeCallBack can be null. 
Except this other bool variables i have assigned false. 🙏 
@masashi-sutou